projects/scion/e2e-testing/src/workbench/popup.e2e-spec.ts

```diff
@@ -541,6 +541,55 @@ test.describe('Workbench Popup', () => {
       await expect(await popup.isPresent()).toBe(true);
       await expect(await popup.isVisible()).toBe(true);
     });
+
+    test('should propagate view context', async ({appPO, workbenchNavigator}) => {
+      await appPO.navigateTo({microfrontendSupport: false});
+
+      // Open target view.
+      const popupTagetViewPage = await workbenchNavigator.openInNewTab(ViewPagePO);
+
+      // Open popup in target view.
+      const popupOpenerPage1 = await workbenchNavigator.openInNewTab(PopupOpenerPagePO);
+      await popupOpenerPage1.selectPopupComponent('popup-opener-page');
+      await popupOpenerPage1.enterCssClass('testee-1');
+      await popupOpenerPage1.enterCloseStrategy({closeOnFocusLost: false});
+      await popupOpenerPage1.enterContextualViewId(popupTagetViewPage.viewId);
+      await popupOpenerPage1.enterPosition({left: 300, top: 300});
+      await popupOpenerPage1.clickOpen();
+
+      const popup1 = appPO.popup({cssClass: 'testee-1'});
```

Comment:
The initial assertions for popup1 should already be covered by a dedicated test for view context?

Reply:
I know, but should also be tested in the context of this test.